diff -ur Django-0.95.1-orig/django/db/backends/ado_mssql/creation.py Django-0.95.1/django/db/backends/ado_mssql/creation.py
--- Django-0.95.1-orig/django/db/backends/ado_mssql/creation.py	2006-05-02 03:31:56.000000000 +0200
+++ Django-0.95.1/django/db/backends/ado_mssql/creation.py	2007-01-24 11:09:12.664664500 +0100
@@ -21,6 +21,5 @@
     'SmallIntegerField': 'smallint',
     'TextField':         'text',
     'TimeField':         'time',
-    'URLField':          'varchar(200)',
     'USStateField':      'varchar(2)',
 }
Tylko w Django-0.95.1/django/db/backends/ado_mssql: creation.py.rej
diff -ur Django-0.95.1-orig/django/db/backends/mysql/creation.py Django-0.95.1/django/db/backends/mysql/creation.py
--- Django-0.95.1-orig/django/db/backends/mysql/creation.py	2006-05-02 03:31:56.000000000 +0200
+++ Django-0.95.1/django/db/backends/mysql/creation.py	2007-01-24 11:09:12.672665000 +0100
@@ -25,6 +25,5 @@
     'SmallIntegerField': 'smallint',
     'TextField':         'longtext',
     'TimeField':         'time',
-    'URLField':          'varchar(200)',
     'USStateField':      'varchar(2)',
 }
Tylko w Django-0.95.1/django/db/backends/mysql: creation.py.rej
diff -ur Django-0.95.1-orig/django/db/backends/oracle/creation.py Django-0.95.1/django/db/backends/oracle/creation.py
--- Django-0.95.1-orig/django/db/backends/oracle/creation.py	2006-05-26 07:37:21.000000000 +0200
+++ Django-0.95.1/django/db/backends/oracle/creation.py	2007-01-24 11:09:12.672665000 +0100
@@ -21,6 +21,5 @@
     'SmallIntegerField': 'smallint',
     'TextField':         'long',
     'TimeField':         'timestamp',
-    'URLField':          'varchar(200)',
     'USStateField':      'varchar(2)',
 }
diff -ur Django-0.95.1-orig/django/db/backends/postgresql/creation.py Django-0.95.1/django/db/backends/postgresql/creation.py
--- Django-0.95.1-orig/django/db/backends/postgresql/creation.py	2006-05-02 03:31:56.000000000 +0200
+++ Django-0.95.1/django/db/backends/postgresql/creation.py	2007-01-24 11:09:12.668664750 +0100
@@ -25,6 +25,5 @@
     'SmallIntegerField': 'smallint',
     'TextField':         'text',
     'TimeField':         'time',
-    'URLField':          'varchar(200)',
     'USStateField':      'varchar(2)',
 }
Tylko w Django-0.95.1/django/db/backends/postgresql: creation.py.rej
diff -ur Django-0.95.1-orig/django/db/backends/sqlite3/creation.py Django-0.95.1/django/db/backends/sqlite3/creation.py
--- Django-0.95.1-orig/django/db/backends/sqlite3/creation.py	2006-05-02 03:31:56.000000000 +0200
+++ Django-0.95.1/django/db/backends/sqlite3/creation.py	2007-01-24 11:09:12.668664750 +0100
@@ -24,6 +24,5 @@
     'SmallIntegerField':            'smallint',
     'TextField':                    'text',
     'TimeField':                    'time',
-    'URLField':                     'varchar(200)',
     'USStateField':                 'varchar(2)',
 }
Tylko w Django-0.95.1/django/db/backends/sqlite3: creation.py.rej
diff -ur Django-0.95.1-orig/django/db/models/fields/__init__.py Django-0.95.1/django/db/models/fields/__init__.py
--- Django-0.95.1-orig/django/db/models/fields/__init__.py	2006-07-28 02:13:10.000000000 +0200
+++ Django-0.95.1/django/db/models/fields/__init__.py	2007-01-24 11:13:37.237199250 +0100
@@ -754,15 +754,19 @@
         val = self._get_val_from_obj(obj)
         return {self.attname: (val is not None and val.strftime("%H:%M:%S") or '')}
 
-class URLField(Field):
+class URLField(CharField):
     def __init__(self, verbose_name=None, name=None, verify_exists=True, **kwargs):
+        kwargs['maxlength'] = kwargs.get('maxlength', 200)
         if verify_exists:
             kwargs.setdefault('validator_list', []).append(validators.isExistingURL)
-        Field.__init__(self, verbose_name, name, **kwargs)
+        CharField.__init__(self, verbose_name, name, **kwargs)
 
     def get_manipulator_field_objs(self):
         return [forms.URLField]
 
+    def get_internal_type(self):
+        return "CharField"
+
 class USStateField(Field):
     def get_manipulator_field_objs(self):
         return [forms.USStateField]
Tylko w Django-0.95.1/django/db/models/fields: __init__.py~
Tylko w Django-0.95.1/django/db/models/fields: __init__.py.orig
Tylko w Django-0.95.1/django/db/models/fields: __init__.py.rej
