--- Django-0.90/django/utils/autoreload.py~	2005-10-31 04:27:13.000000000 +0100
+++ Django-0.90/django/utils/autoreload.py	2005-12-12 15:59:29.951381160 +0100
@@ -39,7 +39,7 @@
         for filename in filter(lambda v: v, map(lambda m: getattr(m, "__file__", None), sys.modules.values())) + reloadFiles:
             if not os.path.exists(filename):
                 continue # File might be in an egg, so it can't be reloaded.
-            if filename.endswith(".pyc"):
+            if filename.endswith(".pyc") and os.path.exists(filename[:-1]):
                 filename = filename[:-1]
             mtime = os.stat(filename).st_mtime
             if filename not in mtimes:
